src/paper/13-q-and-a.ts

```diff
@@ -33,6 +33,8 @@ There’s a \`30%\` chance that I get sick directly from Alice. There’s a \`70
 
 We currently don’t model this effect, and we think that’s pretty safe if (and only if) you and your household members are conscientious about monitoring for symptoms and isolating from one another at the first sign that someone in the house is feeling unwell. Such isolation should include some reduction in contact between the household members that still feel fine, since one or more of them might have become infected by the unwell person before their symptoms appeared.
```

Comment:
And then maybe add a footnote after "feeling unwell" here, to the effect of: "You probably can't do this anywhere near as reliably with someone you don't live with, which is why the discussion here is specific to housemates."